Divine/CLI/CommandLinePackageProcessor.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using Divine.Enums;
+using LSLib.LS;
+
+namespace Divine.CLI
+{
+    internal class CommandLinePackageProcessor
+    {
+        private static readonly CommandLineArguments args = Program.argv;
+
+        public static void Create() => CreatePackageResource();
+
+        [SuppressMessage("ReSharper", "AssignNullToNotNullAttribute")]
+        public static void Extract()
+        {
+            if (CommandLineActions.SourcePath == null)
+            {
+                CommandLineLogger.LogFatal("Cannot extract package without non-null source path");

```

Comment:
A less roundabout way of saying the same thing would be "Cannot extract package without a source path"

Reply:
Okay.